README.md

```diff
@@ -1,5 +1,39 @@
 # Legacy features for FSE
 
-Currently this plugin adds the legacy widgets block and adapt it to work correclty with Polylang.
+Welcome to the Legacy features for FSE repository on GitHub. Here you can browse the source, discuss open issues and keep track of the development.
 
-It also make the legacy menus screen available in the WordPress Appearence admin menu.
+If you are not a developer, we recommend to [download legacy ](https://wordpress.org/plugins/legacy-features-for-fse/) from WordPress directory.
+
+**Todo** Change the link if we change the name of the plugin before publishing on WordPress directory.
+
+## [What this plugin is for?](#legacy-features-for-fse-plugin)
+
+Currently this plugin adds the legacy widgets block in any kind of block editor especially the Full Site Editor(FSE) introduced by WordPress 5.9.
+
+By reintroducing the legacy widgets block, you can use some of the legacy widgets as `Naviagation Menu` for example.
+
+However with blocks theme, like Twenty Twenty-Two, you can't update menus anymore because this kind of theme doesn't support menus.
+
+This is why this plugin reintroduce the WordPress admin menus to be able to manage menus you want to use with the legacy widgets block.
```

Comment:
```suggestion
This is why this plugin reintroduce the WordPress admin menus to be able to manage menus you want to use with the legacy widgets block.

This also allow to use external plugins legacy widgets which might not have already been converted to blocks.
```

Maybe it's an interesting feature to mention. Especially  regarding Polylang?

Reply:
I prefer not to quote Polylang even if we decided to develop it for Polylang at the beginning especially because it could be use with other third party plugins. So the sentence you suggested is perfect for me 😉

Done in 4296ebda7ec4571affb94a4760bd1ee1ffadc912